crates/sc-subspace-block-relay/src/consensus.rs

```diff
@@ -13,19 +13,20 @@ use sc_client_api::{BlockBackend, HeaderBackend};
 use sc_network::request_responses::{IncomingRequest, OutgoingResponse, ProtocolConfig};
 use sc_network::types::ProtocolName;
 use sc_network::{OutboundFailure, PeerId, RequestFailure};
-use sc_network_common::sync::message::{BlockAttributes, BlockData, BlockRequest, FromBlock};
+use sc_network_common::sync::message::{
+    BlockAttributes, BlockData, BlockRequest, Direction, FromBlock,
+};
 use sc_network_sync::block_relay_protocol::{
     BlockDownloader, BlockRelayParams, BlockResponseError, BlockServer,
 };
 use sc_service::SpawnTaskHandle;
 use sc_transaction_pool_api::{InPoolTransaction, TransactionPool, TxHash};
 use sp_runtime::generic::BlockId;
-use sp_runtime::traits::{Block as BlockT, Header};
+use sp_runtime::traits::{Block as BlockT, Header, NumberFor, One, Zero};
 use sp_runtime::Justifications;
-use std::num::NonZeroUsize;
+use std::num::{NonZeroU32, NonZeroUsize};
 use std::sync::Arc;
 use std::time::{Duration, Instant};
-use tracing::{debug, info, trace, warn};
```

Comment:
Why removing import? To my eyes it was a bit nicer before.

Reply:
It is much easier to search for logs and adjust levels if needed, if they are under same prefix

Reply:
It is really not, we don't have anything that is `warn!` except log macro, so I don't understand how it is easier or harder to search, but it is certainly a bit harder to read fully qualified names unnecessarily and we generally use imports for these everywhere else.